requirements/docs.txt

```diff
@@ -79,7 +79,7 @@ packaging==23.1
     #   mkdocs
 pathspec==0.11.1
     # via black
-platformdirs==3.4.0
+platformdirs==3.5.0
```

Comment:
Without this, `make install` complained about conflicting requirements, since `tests.txt` already has `platformdirs==3.5.0`. So I updated this manually. Using `pip-compile` made some other changes I wasn't sure about.